stdlib/public/runtime/Heap.cpp

```diff
@@ -57,6 +61,13 @@ using namespace swift;
 static_assert(_swift_MinAllocationAlignment > MALLOC_ALIGN_MASK,
               "Swift's default alignment must exceed platform malloc mask.");
 
+#if defined(__APPLE__)
+static inline malloc_zone_t *DEFAULT_ZONE() {
+  static malloc_zone_t *z = SWIFT_LAZY_CONSTANT(malloc_default_zone());
```

Comment:
The `static` and `SWIFT_LAZY_CONSTANT` create two levels of lazy initialization; we only need one of them IIUC.

Reply:
Good catch! This shouldn't have the `static`.